terraform/tools/terragrunt.hcl

```diff
@@ -17,10 +17,10 @@ generate "tools_tfvars" {
   oidc_idir_idp_client_id = "fsa-cognito-idir-dev-4088"
   oidc_idir_idp_issuer = "https://dev.loginproxy.gov.bc.ca/auth/realms/standard"
   oidc_bceid_business_idp_client_id = "fsa-cognito-b-ce-id-business-dev-4090"
-  aws_security_group_data = "Data_sg"
+  aws_security_group_data = "fam_data_sg"
```

Comment:
I wonder where we set the "Data_sg" and "App_sg" before? or are they default AWS security groups? And where could I find the network security in AWS console?
<img width="1164" alt="Screen Shot 2023-09-11 at 10 55 43 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/26b5c5ab-3fb9-4b7b-85ef-c95175b3ebb0">

Last question, do we change this for dev/test/prod as well?

Reply:
Answer from the documentation: "Data_sg" and "App_sg" are ASEA-supplied network security groups